src/SourceGenerators/Uno.UI.SourceGenerators/Helpers/HashBuilder.cs

```diff
@@ -28,25 +37,32 @@ public static string BuildIDFromSymbol(ITypeSymbol typeSymbol)
 		/// <param name="input">The string to hash</param>
 		/// <param name="algoritm">The algorithm to use</param>
 		/// <returns>A hex-formatted string of the hash</returns>
-		public static string Build(string input, HashAlgorithm algoritm = null)
+		public static string Build(string input)
```

Comment:
#### Small improvements

1. We were doing unnecessary null check. We always use SHA256 so I removed the optional `algorithm` parameter
2. We were calling `Math.Min` on every iteration. Now the result is stored in a local

#### Larger improvements

We were allocating `StringBuilder` on every call. Then on every iteration we allocate a string of Length 2 containing the hex representation of the current byte.
Finally, at the end we allocate a string from the `StringBuilder`.
Now, we only allocate the final string, and use static fields for lookup.